ruff_lsp/server.py

```diff
@@ -95,6 +97,9 @@
 
 TOOL_MODULE = "ruff.exe" if sys.platform == "win32" else "ruff"
 TOOL_DISPLAY = "Ruff"
+# Don't force users to upgrade ruff if they don't use the linter
+VERSION_REQUIREMENT_LINTER = SpecifierSet(">=0.0.274,<0.2.0")
```

Comment:
Why 0.0.274? I think we can support much older versions.

Reply:
that was the version in pyproject.toml

Reply:
Let's change this to something much lower. There's no reason not to support older versions, there's nothing stopping us. (We can change it in `pyproject.toml` too if you'd like.)

`--force-exclude` was added in v0.0.189, so lets set that as the minimum IMO.

Reply:
fxied